lib/gems/pending/util/mount/miq_generic_mount_session.rb

```diff
@@ -29,18 +29,18 @@ def logger
     @logger ||= $log.nil? ? :: Logger.new(STDOUT) : $log
   end
 
-  def runcmd(cmd_str, args = {})
-    self.class.runcmd(cmd_str, args)
+  def run_command(cmd_str, args = {})
```

Comment:
I renamed this because it makes it easier to search for the callers when the callers aren't named the same.

Also, in a follow up PR I'm going to remove this anyway, because the only callers to run_command are internal, and _ALL_ of them only do `mount` or `unmount`.  IMO, we should then just create a private `mount` method and a private `unmount` method.

Reply:
https://github.com/ManageIQ/manageiq-gems-pending/pull/511 .  Note I built that PR indepedently, so one of these PRs will conflict with the other, but that's ok.